nautobot/dcim/templates/dcim/inc/location_hierarchy.html

```diff
@@ -1,5 +1,6 @@
 {% load helpers %}
 {% if location is not None %}
+{% with site=location.base_site ancestors=location.ancestors %}
```

Comment:
Although `base_site` and `region` are gone in 2.0, it looks like 2.0 could still benefit from `with ancestors=location.ancestors` to make it so that we would only generate that list once instead of twice. Would you be willing to open the equivalent `develop` PR?